samples/dashboards/sync/sync-highlight-options/demo.js

```diff
@@ -0,0 +1,92 @@
+const board = Dashboards.board('container', {
+    dataPool: {
+        connectors: [{
+            id: 'VegeTable',
```

Comment:
Small details, but essentially its: `vegatables`. Do we need capitalics here?

Reply:
I've just copied the connector & data from different demo, I can fix that.